certified-connectors/HiveCPQ - Product configurator/apiDefinition.swagger.json

```diff
@@ -9152,241 +9663,77 @@
           "type": "string",
-          "description": "ID of the price catalog used for this company."
+          "description": "Last name of the contact."
         },
-        "storeDiscountPercentage": {
-          "type": "number",
-          "format": "float",
-          "description": "The discount percentage applied to product store items in a project of this company.",
-          "minimum": -100,
-          "maximum": 100
+        "email": {
+          "type": "string",
+          "description": "E-mail address of the contact."
+        },
+        "salutation": {
+          "type": "string",
+          "description": "Salutation of the contact."
+        },
+        "function": {
+          "type": "string",
+          "description": "Job title of the contact."
+        },
+        "phone": {
+          "type": "string",
+          "description": "Phone number of the contact."
+        },
+        "mobile": {
+          "type": "string",
+          "description": "Mobile phone number of the contact"
+        },
+        "preferredLanguageIso": {
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 5,
+          "description": "Default language of the contact in ISO 639-1 format."
         }
       },
-      "description": "Product store settings of this company."
+      "required": [
+        "firstName",
+        "lastName",
+        "email"
+      ],
+      "description": "General info about the contact."
     },
-    "requestCompanyDefaultProjectSettings": {
+    "distributorContactSalesConditions": {
       "type": "object",
       "properties": {
-        "defaultVatConditionCode": {
+        "vatNumber": {
           "type": "string",
-          "description": "The default VAT condition code to use in projects for this company. Available codes are defined inside your HiveCPQ environment."
+          "description": "VAT number for the contact."
         },
-        "defaultPaymentConditionCode": {
+        "vatCondition": {
           "type": "string",
-          "description": "The default payment condition code to use in projects for this company. Available codes are defined inside your HiveCPQ environment."
+          "description": "The default VAT condition code for this contact. Available codes are defined inside your HiveCPQ environment."
         },
-        "salesPricingStrategy": {
+        "deliveryCondition": {
           "type": "string",
-          "description": "The default pricing strategy to use in projects for this company.",
-          "enum": [
-            "MARKUP",
-            "LIST_PRICE_DISCOUNT"
-          ]
+          "description": "The default delivery condition code for this contact. Available codes are defined inside your HiveCPQ environment."
+        },
+        "paymentCondition": {
+          "type": "string",
+          "description": "The default payment condition code for for this contact. Available codes are defined inside your HiveCPQ environment."
```

Comment:
Please review repeated word "for"

Reply:
The second for has been removed in latest commit.